akd/src/errors.rs

```diff
@@ -45,9 +45,9 @@ impl From<DirectoryError> for AkdError {
     }
 }
 
-impl From<VRFStorageError> for AkdError {
-    fn from(error: VRFStorageError) -> Self {
-        Self::VRFStorageErr(error)
+impl From<VrfError> for AkdError {
```

Comment:
Do we lose any information on the error here? i.e., previously it was `VRF"Storage"Error` now it is only `VrfError`